healpy/__init__.py

```diff
@@ -94,26 +94,29 @@
 from ._sphtools import rotate_alm
 from .rotator import Rotator, vec2dir, dir2vec
 from ._healpy_pixel_lib import UNSEEN
-from .visufunc import (
-    mollview,
-    graticule,
-    delgraticules,
-    gnomview,
-    projplot,
-    projscatter,
-    projtext,
-    cartview,
-    orthview,
-    azeqview,
-)
-from .zoomtool import mollzoom, set_g_clim
+try:
+    from .visufunc import (
+        mollview,
+        graticule,
+        delgraticules,
+        gnomview,
+        projplot,
+        projscatter,
+        projtext,
+        cartview,
+        orthview,
+        azeqview,
+    )
+    from .zoomtool import mollzoom, set_g_clim
+    from .newvisufunc import projview, newprojplot
+except:
+    pass
```

Comment:
If the import fails for any reason other than Matplotlib being missing, then this should raise an exception.

```suggestion
try:
    import matplotlib
except ImportError:
    pass
else:
    from .visufunc import (
        mollview,
        graticule,
        delgraticules,
        gnomview,
        projplot,
        projscatter,
        projtext,
        cartview,
        orthview,
        azeqview,
    )
    from .zoomtool import mollzoom, set_g_clim
    from .newvisufunc import projview, newprojplot
```